Installer.cls

```diff
@@ -15,15 +15,22 @@ XData PM [ XMLNamespace = INSTALLER ]
 <Manifest>
 <Default Name="MODDIR" Dir="${MGRDIR}" />
 <Default Name="SRCDIR" Dir="${MODDIR}src" />
+<Default Name="CORELIB" Value="CORELIB"/>
 <Default Name="REGISTRY" Value="https://pm.community.intersystems.com" />
 <Default Name="TRACKINGID" Value="UA-126752118-1" />
 
+<Namespace Name="${CORELIB}" Code="${CORELIB}" Data="${CORELIB}" Create="yes" Ensemble="0">
```

Comment:
Should not create a namespace. Should also only create the database if it does not already exist

Reply:
@isc-kiyer  Is there a way to only create the database by generating according to manifest? https://docs.intersystems.com/iris20221/csp/docbook/DocBook.UI.Page.cls?KEY=GCI_manifest#GCI_manifest_tags_database the doc indicates that <database> can only be used inside a <configuration>, which has to be inside a <namespace>. <namespace> has to be an existing ns or create a new one. Maybe we can check & create in setup() with code but not sure if that's a good idea.

Reply:
How about create the database while in the %SYS namespace with Create set to no?
Unrelated but I don't like the installer manifests because they are no longer maintained with new properties (e.g. no SessionCookieScope and UserCookieScope for Security.Applications). We could discuss removing all of the installer manifest stuff and using the APIs directly later (Config.* and Security.* classes).